src/jsifier.js

```diff
@@ -398,7 +398,7 @@ function JSify(functionsOnly) {
       print(preprocess(read('arrayUtils.js')));
     }
 
-    if (SUPPORT_BASE64_EMBEDDING && !MINIMAL_RUNTIME) {
+    if ((SUPPORT_BASE64_EMBEDDING || FORCE_FILESYSTEM) && !MINIMAL_RUNTIME) {
```

Comment:
How about replacing this specific change with this:
```diff
diff --git a/emcc.py b/emcc.py
index eaf519567..269e5cebb 100755
--- a/emcc.py
+++ b/emcc.py
@@ -1877,6 +1877,10 @@ def phase_linker_setup(options, state, newargs, settings_map):
   else:
     settings.SYSTEM_JS_LIBRARIES.append((0, shared.path_from_root('src', 'library_pthread_stub.js')))
 
+  if settings.FORCE_FILESYSTEM:
+    # enable base64 decoding for file packager data
+    settings.SUPPORT_BASE64_EMBEDDING = 1
+
   if settings.FORCE_FILESYSTEM and not settings.MINIMAL_RUNTIME:
     # when the filesystem is forced, we export by default methods that filesystem usage
     # may need, including filesystem usage from standalone file packager output (i.e.
```
?

Reply:
The only reason I didn't do it that way is because using `SUPPORT_BASE64_EMBEDDING` has side effects which I'm not sure are needed in this case. Specifically, enabling `SUPPORT_BASE64_EMBEDDING` adds some calls in the code to `tryParseAsDataURI`, which wouldn't be called otherwise.

However, if that's OK, then sure,it's easier to just enable `SUPPORT_BASE64_EMBEDDING` :+1:

Reply:
Oh, very good point. Yes, thinking on this more carefully, I agree with you.